File: TransCoder_tokenized_test_set_functions/python/FIND_SUM_ODD_FACTORS_NUMBER.py
def sumofoddFactors ( n ) :
    res = 1
    while n % 2 == 0 :
        n = n //2
    foriinrange(3,int(math.sqrt(n)+1)):
        count=0
        curr_sum=1
        curr_term=1
        whilen%i==0:
            count+=1
            n=n//i
            curr_term*=i
            curr_sum+=curr_term
        res*=curr_sum
    ifn>=2:
        res*=(1+n)
    returnres

File: results/py/FIND_SUM_ODD_FACTORS_NUMBER.cpp
int f_filled ( int n ) {
  int res = 1 ;
  while ( n % 2 == 0 ) n = n / 2 ;
  foriinrange ( 3 , ( int ) sqrt ( n ) + 1 ) {
    int count = 0 ;
    int currSum = 1 ;
    int currTerm = 1 ;
    while ( whilen % i == 0 ) {
      count ++ ;
      n = n / i ;
      currTerm *= i ;
      currSum += currTerm ;
    }
    res *= currSum ;
  }
  ifn >= 2 ) res *= ( 1 + n ) ;
  return returnres ;
}
